### PR TITLE
Add Steve and Mary as docs maintainers

### PR DESCRIPTION
**Pull request by shykes**

### CODE REVIEW

**Comment by shykes** [comment]
Damn you gordon!

**Comment by crosbymichael** [comment]
LGTM

File: MAINTAINERS [hunk 1]
```diff
@@ -588,3 +590,7 @@ made through a pull request.
 	Name = "Vishnu Kannan"
 	Email = "vishnuk@google.com"
 	GitHub = "vishh"
+
+	[people.mary]
+	Name = "Mary Anthony"
+	Email = "mary.anthony@docker.com"
```
Comment (thaJeztah) [inline MAINTAINERS]: Shouldn't there be a GitHub handle?

Comment (thaJeztah) [inline MAINTAINERS]: Also; the list should be in alphabetical order :smile:

File: MAINTAINERS [hunk 2]
```diff
@@ -403,6 +403,8 @@ made through a pull request.
 				"fredlf",
 				"james",
 				"sven",
+				"spf13",
+				"mary"
```
Comment (thaJeztah) [inline MAINTAINERS]: Should this be ordered alphabetically as well? (no sure, and sorry for being pedantic)

**Comment by SvenDowideit** [comment]
LGTM - @fredlf @jamtur01 

mmm, need to find the docs subsystem rules - I presume we don't want to go back to needing 3 LGTM's

**Comment by thaJeztah** [comment]
@SvenDowideit You can create a DOCS-BOT and add it as maintainer. Everytime you LGTM, it'll automatically repeat you :smile_cat:

**Comment by fredlf** [comment]
lgtm

**Comment by jamtur01** [comment]
Awesome! LGTM.

File: MAINTAINERS (hunk 1 above)
Comment (jfrazelle) [inline MAINTAINERS]: @thaJeztah I will fix in followup PR

Comment (thaJeztah) [inline MAINTAINERS]: @jfrazelle Want me to do that? or would it be faster if you did it? :)

Comment (jfrazelle) [inline MAINTAINERS]: I already had a patch fixing something in it with tomlv too, so I just
added it to that, pinged you on it too :)

**Pull request merged**